### PR TITLE
add documentation, test, and driver for WikipediaFeedExample

### PR DESCRIPTION
**Pull request by dguy**

### CODE REVIEW

**Comment by dguy** [comment]
@miguno

File: kafka-streams/src/main/java/io/confluent/examples/streams/WikipediaFeedAvroLambdaExample.java [hunk 1]
```diff
@@ -38,35 +38,95 @@
...
+ * 4) Write some input data to the source topics (e.g. via {@link WikipediaFeedAvroExampleDriver}).
+ * The already running example application (step 3) will automatically process this input data and
+ * write the results to the output topic. The {@link WikipediaFeedAvroExampleDriver} will print the
+ * results from the output topic
+ *
+ * <pre>
+ * {@code
+ * # Here: Write input data using the example driver.  Once the driver has stopped generating data,
+ * # you can terminate it via `Ctrl-C`.
+ * $ java -cp target/streams-examples-3.1.0-SNAPSHOT-standalone.jar io.confluent.examples.streams.WikipediaFeedAvroExample
```
Comment (miguno) [inline kafka-streams/src/main/java/io/confluent/examples/streams/WikipediaFeedAvroLambdaExample.java]: This should be `WikipediaFeedAvroLambdaExample`, not `WikipediaFeedAvroExample`.

File: kafka-streams/src/test/java/io/confluent/examples/streams/WikipediaFeedAvroLambdaExampleTest.java [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    final Map<String, Long> expected = new HashMap<>();
+    expected.put("donna", 3L);
+    expected.put("becca", 2L);
+    expected.put("john", 1L);
+
+    final Map<String, Long> actual = new HashMap<>();
+
+    consumer.subscribe(Collections.singleton(WikipediaFeedAvroExample.WIKIPEDIA_STATS));
+
+    final long timeout = System.currentTimeMillis() + 30000L;
```
Comment (miguno) [inline kafka-streams/src/test/java/io/confluent/examples/streams/WikipediaFeedAvroLambdaExampleTest.java]: Side note: We are suing this timeout pattern a lot now (which is good).  Is there a way we can avoid copy-pasting essentially the same templated code across our test files?

Comment (dguy) [inline kafka-streams/src/test/java/io/confluent/examples/streams/WikipediaFeedAvroLambdaExampleTest.java]: sure - i'll do it as separate tidy-up

**Pull request merged**